### PR TITLE
R2021a blog post

### PR DESCRIPTION
**Pull request by lukicdarkoo**
**Description**
R2021a announcement blog post.

**Documentation**
https://cyberbotics.com/doc/blog/Webots-2021-a-release?version=docs-r2021a

### CODE REVIEW

**Comment by lukicdarkoo** [comment]
@cyberbotics/maintainers Let me know if we can add something else. I have to improve the text and fix the asset images, but it is roughly the content of the post.

File: docs/blog/Webots-2021-a-release.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Version R2021a Released
+
+<p id="publish-data">By Darko Lukic - **XX**th December 2020</p>
+
+---
+
+Webots R2021a is there! And it's packed some new features, improvements and, of course, bug fixes.
```
Comment (omichel) [inline docs/blog/Webots-2021-a-release.md:7]:
```suggestion
Webots R2021a is there! And it's packed with some new features, improvements and, of course, bug fixes.
```

File: docs/blog/Webots-2021-a-release.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+## Camera Image Segmentation
+
+Webots now is ready for your next deep learning project as it now supports [object recognition and segmentation](https://cyberbotics.com/doc/reference/camera#wb_camera_has_recognition)!
+
+%figure "Image Segmentation"
+![Image Segmentation](images/image_segmentation.gif)
+%end
+
+This new functionality reuses the existing camera recognition functionality to classify the objects in the scene based on properties of the object instance.
+This generic and customizablemethod allows the user to apply any type of segmentation: semantic, instance or panoptic seg-mentation.
```
Comment (omichel) [inline docs/blog/Webots-2021-a-release.md:22]:
```suggestion
This generic and customizable method allows the user to apply any type of segmentation: semantic, instance or panoptic segmentation.
```

File: docs/blog/Webots-2021-a-release.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+We are continuing to develop and improve the support for ROS 2 in Webots with now [13 Webots packages for ROS 2](https://github.com/cyberbotics/webots\_ros2).
+This version of Webots is fully compatible with [ROS 2 Foxy Fitzroy](https://index.ros.org/doc/ros2/Releases/Release-Foxy-Fitzroy).
+
+- We are happy to participate to [Google Season of Docs](https://cyberbotics.com/gsod) this year!
+Thanks to the [Soft illusion YouTube](https://www.youtube.com/channel/UCrl9pLcAAKy8wuXkN-on3xQ) channel a serie of video tutorials about ROS2 and Webots is now available.
+Check [the playlist](https://www.youtube.com/playlist?list=PLt69C9MnPchkP0ZXZOqmIGRTOch8o9GiQ) and learn about ROS2 basics, how to create a Webots ROS2 driver, how to do SLAM, or follow AR tags. 
+- We also signficantly enchanged and refactored our documentation for the [webots\_ros2](https://github.com/cyberbotics/webots\_ros2) package.
+References, examples, and tutorials, are now gathered to [a single place](https://github.com/cyberbotics/webots\_ros2/wiki) with a clean overview.
+- We improved performances of the Lidar node by over 7 times.
```
Comment (omichel) [inline docs/blog/Webots-2021-a-release.md:37]: In English, performance is always singular.
```suggestion
- We improved performance of the Lidar node by over 7 times.
```

File: docs/blog/Webots-2021-a-release.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+- The InertialUnit node now returns quaternions making it easier to use in ROS2 nodes.
+
+---
+
+
+## More Assets
+
+In order to improve our asset library, we have added baskets that may be convinient for your next pick-and-place project.
```
Comment (omichel) [inline docs/blog/Webots-2021-a-release.md:45]:
```suggestion
In order to improve our asset library, we have added baskets that may be convenient for your next pick-and-place project.
```

File: docs/blog/Webots-2021-a-release.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+Webots GitHub repository has reached more than 1000 stars!
+This is fantastic news for us as it shows that the Webots community has been growing ahead of our expectations.
+It has been 2 years since we started the exciting open-source journey.
+We are thankful to all contributors and partners who made the Webots open-source dream becomes a reality.
+
+This year will be also remembered by other milestones:
+- Our Discord community is now counting over 1000 members.
+- The YouTube Cyberobotics channel has exceeded 1000 subscribers.
+- The recently released Webots Docker container has yielded almost 5000 downloads.
```
Comment (stefaniapedrazzi) [inline docs/blog/Webots-2021-a-release.md:23]: Could we maybe also add the links to Discord/YouTube and Docker?

```suggestion
- Our Discord community is now counting over 1000 members.
- The YouTube Cyberbotics channel has exceeded 1000 subscribers.
- The recently released Webots Docker container has yielded almost 5000 downloads.
```

File: docs/blog/Webots-2021-a-release.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+## Camera Image Segmentation
+
+Webots now is ready for your next deep learning project as it now supports [object recognition and segmentation](https://cyberbotics.com/doc/reference/camera#wb_camera_has_recognition)!
+
+%figure "Image Segmentation"
+![Image Segmentation](images/image_segmentation.gif)
+%end
+
+This new functionality reuses the existing camera recognition functionality to classify the objects in the scene based on the properties of the object instance.
+This generic and customizable method allows the user to apply any type of segmentation: semantic, instance, or panoptic seg-mentation.
```
Comment (stefaniapedrazzi) [inline docs/blog/Webots-2021-a-release.md:36]:
```suggestion
This generic and customizable method allows the user to apply any type of segmentation: semantic, instance, or panoptic segmentation.
```

File: docs/blog/Webots-2021-a-release.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+Simply setting the recognition property at the object model layer, will generate a semantic segmentation.
+
+---
+
+## ROS 2 & Webots
+
+We are continuing to develop and improve the support for ROS 2 in Webots with now [13 Webots packages for ROS 2](https://github.com/cyberbotics/webots\_ros2).
```
Comment (stefaniapedrazzi) [inline docs/blog/Webots-2021-a-release.md:43]: What is the correct spelling `ROS2` or `ROS 2`?

File: docs/blog/Webots-2021-a-release.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+This version of Webots is fully compatible with [ROS 2 Foxy Fitzroy](https://index.ros.org/doc/ros2/Releases/Release-Foxy-Fitzroy).
+
+- We are happy to participate in [Google Season of Docs](https://cyberbotics.com/gsod) this year!
+Thanks to the [Soft illusion YouTube](https://www.youtube.com/channel/UCrl9pLcAAKy8wuXkN-on3xQ) channel a series of video tutorials about ROS2 and Webots is now available.
+Check [the playlist](https://www.youtube.com/playlist?list=PLt69C9MnPchkP0ZXZOqmIGRTOch8o9GiQ) and learn about ROS2 basics, how to create a Webots ROS2 driver, how to do SLAM, or follow AR tags. 
+- We also significantly enhanced and refactored our documentation for the [webots\_ros2](https://github.com/cyberbotics/webots\_ros2) package.
+References, examples, and tutorials, are now gathered to [a single place](https://github.com/cyberbotics/webots\_ros2/wiki) with a clear overview.
+- We improved the performance of the Lidar node by over 7 times.
```
Comment (stefaniapedrazzi) [inline docs/blog/Webots-2021-a-release.md:51]:
```suggestion
- We improved the performance of the [Lidar](https://cyberbotics.com/doc/reference/lidar) node by over 7 times.
```

File: docs/blog/Webots-2021-a-release.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+
+Webots now is ready for your next deep learning project as it now supports [object recognition and segmentation](https://cyberbotics.com/doc/reference/camera#wb_camera_has_recognition)!
+
+%figure "Image Segmentation"
+![Image Segmentation](images/image_segmentation.gif)
+%end
+
+This new functionality reuses the existing camera recognition functionality to classify the objects in the scene based on the properties of the object instance.
+This generic and customizable method allows the user to apply any type of segmentation: semantic, instance, or panoptic seg-mentation.
+Simply setting the recognition property at the object model layer, will generate a semantic segmentation.
```
Comment (stefaniapedrazzi) [inline docs/blog/Webots-2021-a-release.md:37]:
```suggestion
After setting the recognition property of an object, the object will be immediately displayed in the segmentation image.
```

File: docs/blog/Webots-2021-a-release.md [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+This version of Webots is fully compatible with [ROS 2 Foxy Fitzroy](https://index.ros.org/doc/ros2/Releases/Release-Foxy-Fitzroy).
+
+- We are happy to participate in [Google Season of Docs](https://cyberbotics.com/gsod) this year!
+Thanks to the [Soft illusion YouTube](https://www.youtube.com/channel/UCrl9pLcAAKy8wuXkN-on3xQ) channel a series of video tutorials about ROS2 and Webots is now available.
+Check [the playlist](https://www.youtube.com/playlist?list=PLt69C9MnPchkP0ZXZOqmIGRTOch8o9GiQ) and learn about ROS2 basics, how to create a Webots ROS2 driver, how to do SLAM, or follow AR tags. 
+- We also significantly enhanced and refactored our documentation for the [webots\_ros2](https://github.com/cyberbotics/webots\_ros2) package.
+References, examples, and tutorials, are now gathered to [a single place](https://github.com/cyberbotics/webots\_ros2/wiki) with a clear overview.
+- We improved the performance of the Lidar node by over 7 times.
+- The InertialUnit node now returns quaternions making it easier to use in ROS2 nodes.
```
Comment (stefaniapedrazzi) [inline docs/blog/Webots-2021-a-release.md:52]:
```suggestion
- The [InertialUnit](https://cyberbotics.com/doc/reference/inertialunit) node now returns quaternions making it easier to use in ROS2 nodes.
```

File: docs/blog/Webots-2021-a-release.md [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+---
+
+## Extra Goodies
+
+Webots now supports Python 3.9 on all platforms (Windows, Linux, and macOS).
+
+Devices added during the simulation run are available in the robot controller.
+We believe this feature will be especially interesting for deep learning applications.
+
+We also added [exposure control](camera.md#wb_camera_get_exposure) to the camera node.
```
Comment (stefaniapedrazzi) [inline docs/blog/Webots-2021-a-release.md:90]:
```suggestion
We also added [exposure control](camera.md#wb_camera_get_exposure) to the [Camera](https://cyberbotics.com/doc/reference/camera) node.
```

**Review by stefaniapedrazzi — CHANGES_REQUESTED** [review]
The blog is very good to me.
I don't have anything major to add, just a couple of improvements.

Do I have to add an image for the camera segmentation feature?

File: docs/blog/Webots-2021-a-release.md (hunk 7 above)
Comment (omichel) [reply docs/blog/Webots-2021-a-release.md:43]: I would go for `ROS2`.

**Comment by omichel** [comment]
On a second thought, I believe "ROS 2" is the official spelling see https://index.ros.org/doc/ros2/. I will revert my changes.

File: docs/blog/Webots-2021-a-release.md (hunk 7 above)
Comment (omichel) [reply docs/blog/Webots-2021-a-release.md]: On a second thought, I believe "ROS 2" is the official spelling see https://index.ros.org/doc/ros2/. I will revert my changes.

**Comment by omichel** [comment]
> Do I have to add an image for the camera segmentation feature?

I don't think so, we as already have one with the automobile simulation.

**Comment by stefaniapedrazzi** [comment]
> > Do I have to add an image for the camera segmentation feature?
> 
> I don't think so, we as already have one with the automobile simulation.

I didn't saw it because when loading the page in the rich text mode it is not always uploaded.

**Review by stefaniapedrazzi — APPROVED** [review]
Thank you. It's all fine for me.
Now we just miss the release date.

**Review by stefaniapedrazzi — APPROVED** [review]

**Pull request merged**